### DISCUSSION TITLE
Cache resources exhausted with Illustrator file

### DISCUSSION THREAD

**Discussion by ashishc-opentext** [ImageMagick/ImageMagick#3853] [Digital Image Processing]
**Command to be executed :**
`magick.exe -density 300 -colorspace RGB D:\SLS_v1\ART-66084\ART-66084\Sweet_Sale_2019_Diagonal_Lines_RED.ai[0-0] -alpha remove -sample 1200x1200 -quality 92 D:\SLS_v1\ART-66084\ART-66084\out\Sweet_Sale_2019_Diagonal_Lines_RED-out.jpg`

**Command output :**
`magick.exe: cache resources exhausted `D:/tempIMArea/magick-11796-m8UTsVxQb001' @ error/cache.c/OpenPixelCache/3866.
magick.exe: PDFDelegateFailed `[ghostscript library 9.18] -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pngalpha" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r300x300" -dFirstPage=1 -dLastPage=1 "-sOutputFile=D:/tempIMArea/magick-11796-m8UTsVxQb00%d" "-fD:/tempIMArea/magick-11796Db-fbgxuEPBZ" "-fD:/tempIMArea/magick-117966wbcOKLUUEc_": (null)' @ error/pdf.c/ReadPDFImage/796.`

**Policy.xml configuration :**
`<policymap>
  <policy domain="resource" name="temporary-path" value="D:\\tempIMArea"/>
  <policy domain="resource" name="memory" value="8GiB"/>
  <policy domain="resource" name="area" value="4GiB"/>
  <policy domain="resource" name="disk" value="8GiB"/>
  <policy domain="cache" name="shared-secret" value="passphrase" stealth="true"/>
</policymap>`

**identify -list resource**

`Resource limits:
  Width: 107.374MP
  Height: 107.374MP
  List length: -1
  Area: 4.29497GP
  Memory: 3.92739GiB
  Map: 7.85477GiB
  Disk: 8GiB
  File: 1536
  Thread: 12
  Throttle: 0
  Time: unlimited`

**Comment by snibgo** [comment]
Try with a much smaller density, such as "3". How many pixels are in the output? What is that, multiplied by 100 twice? Do you have enough free memory or disk for that many pixels?
